### ISSUE TITLE
Image corruption with xinerama

### ISSUE THREAD

**Issue by Ionic** [ArcticaProject/nx-libs#681]
![nxagent-xinerama-corruption2](https://user-images.githubusercontent.com/1229578/37502299-6463b732-28d2-11e8-8f28-ecdd69b0e2e9.png)

This was created by dragging the display utility window inside of the session around.

Dragging the `nxagent` window around usually forces a refresh and eliminates the corruption.

**Comment by uli42** [comment]
Have you added your second screen AFTER the x2go session start?

Mihai Moldovan <notifications@github.com> schrieb am Fr., 16. März 2018,
04:30:

**Comment by Ionic** [comment]
Nope, it's been there all the time. The only thing I did was dragging the monitor preferences window around within the session.

What you cannot properly see is that behind that window, there is a MATE control center window which also shows up corrupted.

This is how it looks like after a forced refresh by dragging the nxagent window a bit:
![nxagent-xinerama-corruption3](https://user-images.githubusercontent.com/1229578/37508930-88c2b01a-28f5-11e8-81eb-471ebcfb8d14.png)
